app/src/main/java/ch/admin/bag/dp3t/inform/ThankYouFragment.java

```diff
@@ -27,12 +34,32 @@ public ThankYouFragment() {
 		super(R.layout.fragment_thank_you);
 	}
 
+	@Override
+	public void onCreate(@Nullable Bundle savedInstanceState) {
+		super.onCreate(savedInstanceState);
+		secureStorage = SecureStorage.getInstance(getContext());
+	}
+
 	@Override
 	public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
 		super.onViewCreated(view, savedInstanceState);
 
 		((InformActivity) requireActivity()).allowBackButton(false);
 
+		// Show the onset date in the thank you message
+		TextView thankYouTextView = view.findViewById(R.id.inform_thank_you_text);
+		long onsetDateInMillis = secureStorage.getPositiveReportOnsetDate();
+		if (onsetDateInMillis > 0L) {
+			String formattedDate = DateUtils.getFormattedDateWrittenMonth(onsetDateInMillis);
```

Comment:
This method depends on the device timezone which might lead to a wrong date being displayed!